packages/core/src/utils/createHash.ts

```diff
@@ -0,0 +1,6 @@
+import { anyToString } from './stringUtils';
+import { createHash as createHashRaw } from 'crypto';
+
+export default function createHash(content: any) {
+  return createHashRaw('sha256').update(anyToString(content)).digest('hex');
+}
```

Comment:
We can use this in `cache.ts`, inside of `getSchemaHash()` to avoid duplication.